src/modules/machine/actions/MachineActions.test.tsx

```diff
@@ -0,0 +1,69 @@
+import { render, fireEvent, screen } from "@testing-library/react";
+import { BrowserRouter } from "react-router-dom";
+import { Provider } from "react-redux";
+import store from "../../core/store/";
+import MachineActions from "./MachineActions";
+
+describe("MachineActions component", () => {
+  it("should render MachineActions component", () => {
+    render(
+      <BrowserRouter>
+        <Provider store={store}>
+          <MachineActions />
+        </Provider>
+      </BrowserRouter>
+    );
+  });
+
+  it("should add a machine action", () => {
+    render(
+      <BrowserRouter>
+        <Provider store={store}>
+          <MachineActions />
+        </Provider>
+      </BrowserRouter>
+    );
+
+    const addButton = screen.getByRole("button", { name: "enqueue" });

```

Comment:
It's better to add to button data-testid and search it by this attribute

https://testing-library.com/docs/queries/bytestid/